greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-6288: Fix Share buttons on EN Form beta

Ref: https://jira.greenpeace.org/browse/PLANET-6288

- Fix links for Share buttons
- Fix icons in the editor
- Fix title/description extracted from the wrong post ID
- Extract ShareButtons to components for reusability

## Test

- Enable beta blocks in the admin in "Planet 4 > Features"
- Create a page, add a block "EN Form (beta)"

Share buttons appear in the "Thank you" panel of the EN form.

![Screenshot from 2021-09-21 10-59-44](https://user-images.githubusercontent.com/617346/134142792-7673808b-ba22-421d-ae18-58d6acff6e20.png)

- In the post editor, this panel is accessible through a button on the EN form block :
![Screenshot from 2021-09-21 10-55-02](https://user-images.githubusercontent.com/617346/134142268-f8e51ce4-11e8-4ef8-a0c4-330e30ce09b4.png)

- On the frontend, if you have the [React dev tools](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/) installed, you can enable it by modifying the hooks state of the block, from `"signin"` to `"thankyou"`:
![Screenshot from 2021-09-21 10-55-53](https://user-images.githubusercontent.com/617346/134142507-775b80a4-b59d-4007-a668-0f0e0ca7a0d8.png)

This is a port of the twig template [`share_buttons.twig`](https://github.com/greenpeace/planet4-master-theme/blob/master/templates/blocks/share_buttons.twig) and should behave the same.
